packages/relay/tests/lib/utils.spec.ts

```diff
@@ -45,4 +47,44 @@ describe('Utils', () => {
       });
     }
   });
+
+  describe('createSyntheticLogsBloom and checkInLogsBloom', () => {
```

Comment:
Add tests for:
- `createSyntheticLogsBloom` with:
  - `undefined`/`null`/`empty` address string
  - invalid topics array: `[...topics, "nonHexString"]`/`[...topics, 1]`/`[...topics, {}]`
- `checkInLogsBloom` with:
  - `undefined`/`null`/`empty` item string
  - `undefined`/`null`/`empty` bit vector string